app/models/user.rb

```diff
@@ -20,6 +20,11 @@ class User < ActiveRecord::Base
   has_many :outgoing_delegations, class_name: 'ApprovalDelegate', foreign_key: 'assigner_id'
   has_many :outgoing_delegates, through: :outgoing_delegations, source: :assignee
 
+  has_many :incoming_delegations, class_name: 'ApprovalDelegate', foreign_key: 'assignee_id'
```

Comment:
surprised code climate isn't yelling about single quotes. Seems like that integration is a bit flaky these days...